lms/djangoapps/grades/receivers.py

```diff
@@ -0,0 +1,139 @@
+"""
+Grades related signals.
+"""
+from django.conf import settings
+from logging import getLogger
+from opaque_keys.edx.locator import CourseLocator
+from opaque_keys.edx.keys import UsageKey
+
+from .signals import SCORE_CHANGED
+from .transformer import GradesTransformer
+from .new.subsection_grade import SubsectionGradeFactory
+from openedx.core.djangoapps.content.block_structure.api import get_block_structure_manager
+from student.models import user_by_anonymous_id
+
+log = getLogger(__name__)
+
+
+def submissions_score_set_handler(sender, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Ah.. in fact, I just found [a blog](https://chriskief.com/2014/02/28/django-1-7-signals-appconfig/) that uses a `signals` submodule in their example.  See the section that starts with "Finally, here’s an example signal and handler from my signals submodule".